src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -219,15 +219,31 @@ public static bool IsConsumptionSku(this IEnvironment environment)
         /// <returns><see cref="true"/> if running in the FlexConsumption Sku, false otherwise.</returns>
         public static bool IsFlexConsumptionSku(this IEnvironment environment)
         {
-            string value = environment.GetEnvironmentVariable(AzureWebsiteSku);
-            if (string.Equals(value, ScriptConstants.FlexConsumptionSku, StringComparison.OrdinalIgnoreCase))
+            string sku = environment.GetEnvironmentVariable(WebsiteSkuName);
+
+            // Currently set the sku to Dynamic in FPS
+            // TODO: Remove this once FPS sets the value to FlexConsumption
+            if (string.Equals(sku, ScriptConstants.DynamicSku, StringComparison.OrdinalIgnoreCase))
             {
                 return true;
             }
+            return string.Equals(sku, ScriptConstants.FlexConsumption, StringComparison.OrdinalIgnoreCase);
+        }
+
+        public static bool IsConsumptionV1(this IEnvironment environment)
```

Comment:
I'd rename this to IsLinuxConsumptionV1 if that's what you're trying to capture?